src/cmd/deploy.ts

```diff
@@ -372,7 +372,8 @@ export default class DefenderDeploy {
           name: contract.name,
           network: match.network,
           address: match.address,
-          abi: contract.abi && JSON.stringify(JSON.parse(contract.abi)),
+          abi:
+            contract.abi && JSON.stringify(typeof contract.abi === 'string' ? JSON.parse(contract.abi) : contract.abi),
```

Comment:
Maybe we can turn this into a helper function "formatABI" since we're using it in a few places.

Reply:
Added helper function to `utils/index.ts`